### PR TITLE
Updated workflow for rhel6 to rhel7 migration.

### PR DESCRIPTION
**Pull request by sthirugn**

### CODE REVIEW

File: docs/satellite-clone.md [hunk 1]
```diff
@@ -23,6 +23,14 @@ After cloning:
 
 RHEL6 to RHEL7 migration:
   - You can clone RHEL 6 backup data to a RHEL 7 machine.  In this case, you must update the variable `rhel_migration` to `true` as explained later in this document.
+  - Typical Migration workflow:
```
Comment (johnpmitsch) [inline docs/satellite-clone.md]: This is good to document the production clone workflow, but this isn't exclusive to a RHEL migration. We can put this into a separate section called "Production clone workflow" or similar

**Comment by sthirugn** [comment]
@johnpmitsch updated

**Comment by johnpmitsch** [comment]
@sthirugn Looks good. The formatting seems off here, I also think a few notes about what a production workflow means - as in, This is a workflow to transition your environment from a working Satellite to a cloned Satellite.

**Comment by sthirugn** [comment]
ah missed the formatting, updated now @johnpmitsch

**Comment by sthirugn** [comment]
thank you @johnpmitsch

**Pull request merged**